tests/e2e-tests/src/recursor/security/scenarios.rs

```diff
@@ -40,7 +40,8 @@ fn tx_id_validation_test() -> Result<()> {
     let _leaf_ns = leaf_ns.start()?;
 
     thread::sleep(Duration::from_secs(2));
-    let a_settings = *DigSettings::default().recurse().authentic_data();
+
+    let a_settings = *DigSettings::default().recurse().timeout(7);
```

Comment:
This removes the `authentic_data()` call. Is that intentional? If so, maybe in a different commit?